Cython/Compiler/Scanning.py

```diff
@@ -301,7 +301,7 @@ class PyrexScanner(Scanner):
     #  compile_time_eval  boolean  In a true conditional compilation context
     #  compile_time_expr  boolean  In a compile-time expression context
 
-    def __init__(self, file, filename, parent_scanner=None,
+    def __init__(self, file, filename, fault_tolerant, parent_scanner=None,
```

Comment:
Add `fault_tolerant=False` as the last parameter (otherwise it'd change the api and break callers).